middleware/authentication_test.go

```diff
@@ -43,7 +43,7 @@ func TestHTTPBasicAuthenticationMW(t *testing.T) {
 	// Missing authorization token.
 	{
 		var w = httptest.NewRecorder()
-		mockLogger.EXPECT().Info("Authorization Error", "Missing Authorization header").Return(nil).Times(1)
+		mockLogger.EXPECT().Info(context.TODO(), "Authorization Error", "Missing Authorization header").Return(nil).Times(1)
```

Comment:
I think we don't care of the value of the context here... May be you should use gomock.Any() so that you don't have to update your tests if you change the value you provided in authentication.go (ex: if you decide to provide context.Background() rather than context.TODO())